### ISSUE TITLE
Don't generate schemas for models without swagger:model annotation

### ISSUE THREAD

**Issue by mcandre** [go-swagger/go-swagger#1079]
When generating models with `-m`, don't generate models for things that don't actually have the `// swagger:model` annotation, as this can lead to a lot of junk schemas

**Comment by evillemez** [comment]
Agreed... it took me a little bit to realize what was happening, I just assumed that was how it worked, but apparently not.  At the moment, I have internal types that are bleeding out into the documentation as a result.

**Comment by fredbi** [comment]
Does the swagger:ignore annotation solve the issue/answer the question?
From P.R: New swagger:ignore annotation to explicitly ignore extraneous models #1152

**Comment by pault84** [comment]
@fredbi i don't think so. i'm getting documentation for a timestamp from another package that's not mine. because i use -m. i can't really go and change that package with :ignore

**Comment by fredbi** [comment]
OK. Just wanted to clarify the situation regarding issues, questions etc... Good to get some feedback.
This problem is relatively well explained in #796. 
Unfortunately, previous work on this has been left unfinished.
